src/clj/rems/css/styles.clj

```diff
@@ -541,7 +541,7 @@
                    :flex-direction "row"
                    :justify-content "space-between"
                    :min-width "100%"}
-    [:nav {:flex 1}]]
+    ["> .navbar:first-child" {:flex 1}]]
```

Comment:
Is it possible to avoid this complicated selector?

Reply:
What do you propose? Some options are adding a new class to the div and using that, or moving the flex:1 to a style tag.

Reply:
Maybe the child component could have a class (e.g. `d-flex`), that would be clearer.

Reply:
Done